packages/python/plotly/plotly/express/_chart_types.py

```diff
@@ -461,6 +461,11 @@ def box(
     """
     In a box plot, rows of `data_frame` are grouped together into a
     box-and-whisker mark to visualize their distribution.
+    Each box spans from quartile 1 (Q1) to quartile 3 (Q3). The
```

Comment:
could you please add a blank line before this sentence? The convention is to have a 1-line sentence to start the docstring, then if needed explanations in another paragraph.